homeassistant/components/tellduslive/__init__.py

```diff
@@ -70,8 +70,8 @@
 
     client = TelldusLiveClient(hass, entry, session)
     hass.data[DOMAIN] = client
-
-    await client.update()
+    hass.async_add_executer_job(async_add_hubs(hass, client, entry.entry_id))
```

Comment:
No, these are coroutines and must be scheduled on the event loop. Either await them or use `hass.async_create_task`. Usually `await` is fine. When we use `await` we wait until they are scheduled and return a result. When we schedule a task, we don't wait for the result.

Reply:
We do not need to wait for the update so I changed it to `hass.async_create_task`.